discojs/secAggREADME.md

```diff
@@ -0,0 +1,20 @@
+# Secure Multiparty Aggregation for Decentralized Training
+
+### Algorithm description
+Additive sharing (you can read more about this algorithm [HERE](https://www.geeksforgeeks.org/additive-secret-sharing-and-share-proactivization-using-python/?ref=rp)
+) is implemented with each client's gradients when the gradient is sent to all other clients connected in this round of multiparty training. The algorithm has four steps:
```

Comment:
for generality, say model updates / models (can be both in the future), instead of gradients